splunkapp/generate_wazuh_splunk_app.sh

```diff
@@ -44,7 +45,7 @@ build_package() {
     # Build the Docker image
     docker build -t ${CONTAINER_NAME} ./Docker/
     # Run Docker and build package
-    docker run -t --rm -v "${TMP_DIRECTORY}":/pkg -v "${DESTINATION}":/wazuh_splunk_app ${CONTAINER_NAME} ${wazuh_version} ${splunk_version} ${REVISION} || exit 1
+    docker run -t --rm -v "${TMP_DIRECTORY}":/pkg -v "${DESTINATION}":/wazuh_splunk_app ${CONTAINER_NAME} ${wazuh_version} ${splunk_version} ${REVISION} ${cheksum}|| exit 1
```

Comment:
Why there are variables in lowercase and other ones in uppercase? If in every script we are using uppercase to define the variables, follow that "standard".

In addition: Typo => `cheksum`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/ba5843f8aac9201fe518b0d257c88d6493431ec2 and https://github.com/wazuh/wazuh-packages/pull/192/commits/5de766ec4dd5cddd21718c5897f9431117361be8

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/192/commits/ba5843f8aac9201fe518b0d257c88d6493431ec2 and https://github.com/wazuh/wazuh-packages/pull/192/commits/5de766ec4dd5cddd21718c5897f9431117361be8